extension/content.js

```diff
@@ -117,9 +133,14 @@ function moveVotes() {
 
 		const isUp = commentIsUseless('upvote', el);
 		const isDown = commentIsUseless('downvote', el);
-		const commenter = $(el).closest('.js-comment-container').find('.author').get(0).innerHTML;
 
 		if (isUp || isDown) {
+			// grab avatar and wrapping a tag for commenter
+			const commenterHTML = $(el).closest('.js-comment-container').find('a').get(0);
```

Comment:
No, this gives you the node as an object and I want the element as a string to add to the `Set()`. I tried this approach originally and it didn't work out.